src/App.js

```diff
@@ -1,25 +1,80 @@
-import React from 'react';
+import React, { Component } from 'react';
 
 import './App.scss';
+import Clock from './components/Clock';
 
-const App = () => {
-  setInterval(() => {
-    const date = new Date();
-
-    // eslint-disable-next-line
-    console.log(date.toLocaleTimeString());
-  }, 1000);
-
-  return (
-    <div className="App">
-      <h1>React clock</h1>
-      <p>
-        Current time:
-        {' '}
-        {/* Print the time here instead of DevTools */}
-      </p>
-    </div>
-  );
-};
+class App extends Component {
+  state = {
+    isClockVisible: true,
+    clockName: 1,
+  }
+
+  componentDidMount() {
+    setInterval(() => {
+      const date = new Date();
+
+      // eslint-disable-next-line
+      console.log(date.toLocaleTimeString());
+    }, 1000);
+  }
+
+  hideClock = () => {
+    this.setState({
+      isClockVisible: false,
+    });
+  }
+
+  showClock = () => {
+    this.setState({
+      isClockVisible: true,
+    });
+  }
+
+  renameClock = () => {
+    this.setState({
+      clockName: Math.random(),
+    });
+  }
+
+  render() {
+    const currClockName = this.state.clockName;
+
```

Comment:
Почему бы тут не сделать такую деструктуризацию
```suggestion
    const { isClockVisible, clockName } = this.state;

```

Reply:
Done